vm/src/stdlib/builtins.rs

```diff
@@ -256,6 +256,15 @@ mod builtins {
                     ));
                 }
 
+                // Check for newlines, tabs, or whitespaces at start
```

Comment:
Thanks! Though there's not really a need to manually remove them, you can use `trim_start()` on source before we create the `PyStr` from it, i.e `vm.ctx.new_str(source.trim_start())`.

Note that CPython currently ignores tabs and spaces only, don't think this is really an issue since the lexer ignores those either way.